README.md

```diff
@@ -16,9 +16,13 @@ for the purposes of speed and efficiency, reliability, scale, and collaboration.
 ## What is RSE-ops?
 
 If DevOps is the intersection of  "Developer" and "Operations," then how does this concept map to high performance computing? The answer is RSE-ops, or the intersection of Research Software Engineering and Operations. Research Software Engineers (RSEs) are the individuals writing code for scientific software, and supporting resarchers to use codes on high performance computing systems.
-RSE-ops, then, generally refers to best practices for development and operations of scientific software, which typically happens in a high performance computing environment. A comparison can be made to Cloud Native Devops [^2], which is a similar term that has emerged around Cloud Native Computing. It refers to the application of DevOps practices to the cloud.
+RSE-ops, then, generally refers to best practices for development and operations of scientific software, which typically happens in a high performance computing environment.
 
-The role of Research Software Engineer, or RSE, has been emerging in the last decade, and due to the hybrid nature of working environments across cloud, and HPC, DevOps practices are logically being adopted. In the same way that Cloud Native has defined development approaches that focus on maximizing a particular set of resources for building, testing, and running software for the cloud, RSE-ops can do the same for HPC. For the purposes of this document, we will consider modern DevOps to to be Cloud Native, and use the two interchangeably. We will also drop the "Dev" from the parallel RSE DevOps role, as the term "Research Software Engineering" encompasses it. RSE-ops, then, appropriately refers to best practices for ensuring the same reiability, scale, and collaboration, and software engineering for research codes. We may not always be running a scaled web service, but we might be running scaled jobs on a manager, profiling performance, or testing software in development.
+> I have wondered if this is true -- aren't there lots of non-HPC RSE's?
```

Comment:
yeah I was actually surprised how close the two were, but it makes sense since RSE's are needed frequently but not always for research computing.  I just don't think we should tie them directly together -- as there's some nuance there.

Reply:
Agree.